Doc/library/asyncio-eventloop.rst

```diff
@@ -59,9 +59,8 @@ an event loop:
    instead of using these lower level functions to manually create and close an
    event loop.
 
-   .. deprecated:: 3.12
-      Deprecation warning is emitted if there is no current event loop.
-      In some future Python release this will become an error.
+   .. versionchanged:: 3.14
+      Raises a :exc:`RuntimeError` if there is no set event loop.
```

Comment:
We perhaps should use something like "by default" here, because custom policy can still create a new event loop.

Also, "if there is no set event loop" is not accurate. It is rather "if there is no running event loop and the current event loop was not set".

Reply:
if there is running loop, doesn't it mean that current loop is set?

Reply:
This is something not clear terminology. The current event loop can be set implicitly via `set_even_loop()` (this does not make it a running loop) or it can be a running event loop, even if it was not set by `set_even_loop()`. If the longer description is confusing, then maybe use "if there is no current event loop"?

Reply:
> "if there is no current event loop"?

That sounds good, changed it